app/project_type/directory.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, project_directory, config=None, job_name=None, build_project_
         self.project_directory = os.path.abspath(project_directory)
         self._logger.debug('Project directory is {}'.format(project_directory))
 
-    def _setup_build(self):
+    def _fetch_project(self):
         check_command = 'test -d "{}"'.format(self.project_directory)
         output, exit_code = self.execute_command_in_project(check_command, cwd='/')
         if exit_code != 0:
```

Comment:
I agree that we should change the name of this method to something besides setup_build. I'm wondering about using the verb "fetch" -- it is a bit counterintuitive that "fetch_project" for directory type just tests if the directory is there. Maybe something like "prepare_project" would be more apt?

I'm not too concerned about this actually. "fetch_project" is still better and directory project types aren't really commonly used anyway. :)